src/core/lib/security/credentials/tls/grpc_tls_credentials_options.h

```diff
@@ -45,11 +45,19 @@ struct grpc_tls_key_materials_config
   void set_pem_root_certs(grpc_core::UniquePtr<char> pem_root_certs) {
```

Comment:
Is this API used anywhere? If not can we delete it?

Reply:
This function is used in a function with the same name in C++ wrapper.

Reply:
nit. Please add a TODO to remove it later.